docs/source/deployment/CORS_Headers.rst

```diff
@@ -0,0 +1,95 @@
+========================================
+Add CORS Headers to MOTECH Configuration
+========================================
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Add CORS support to Apache server
```

Comment:
I feel like this docs needs an intro. What are these CORS headers and why do we need them?